### ISSUE TITLE
Fire Emblem - Font issues

### ISSUE THREAD

**Issue by MP2E** [MiSTer-devel/GBA_MiSTer#24]
Some of the fonts are drawn incorrectly on MiSTer currently. I am using Integer Scale and the filter named "No Interpolation" to make sure no filters are causing this

![Screenshot from 2019-12-10 23-11-12](https://user-images.githubusercontent.com/167708/70599927-476a7600-1ba3-11ea-8c71-ea6fd7a314f1.png)
![Screenshot from 2019-12-10 23-11-37](https://user-images.githubusercontent.com/167708/70599929-48030c80-1ba3-11ea-9438-063e1dfc3fa1.png)
![Screenshot from 2019-12-10 23-12-01](https://user-images.githubusercontent.com/167708/70599930-48030c80-1ba3-11ea-9d05-45d1110582ba.png)
![Screenshot from 2019-12-10 23-12-08](https://user-images.githubusercontent.com/167708/70599931-48030c80-1ba3-11ea-800b-13c9e878dedd.png)

Here's my Game Boy Player via GBIHF and an OSSC for comparison

![Screenshot from 2019-12-10 23-15-14](https://user-images.githubusercontent.com/167708/70599971-6537db00-1ba3-11ea-9c31-3bff110c50db.png)
![Screenshot from 2019-12-10 23-15-27](https://user-images.githubusercontent.com/167708/70599973-6537db00-1ba3-11ea-9619-f0c2d05cc8c9.png)
![Screenshot from 2019-12-10 23-15-33](https://user-images.githubusercontent.com/167708/70599975-6537db00-1ba3-11ea-9373-796b90f419c2.png)
![Screenshot from 2019-12-10 23-15-47](https://user-images.githubusercontent.com/167708/70599976-6537db00-1ba3-11ea-9eb8-d68ef85a8100.png)

**Comment by sorgelig** [comment]
Note: integer scaling is only integer for vertical scale. Horizontal scale is always non-integer as it's impossible to accomplish.

**Comment by RobertPeip** [comment]
i have seen this bug on my hardware, i tried to reproduce and now it's gone...So it's no static drawing effect, more likely some cpu miscalculation under certain conditions...

**Comment by MP2E** [comment]
I noticed this also happens in Fire Emblem: The Sacred Stones, and there happens to be a decompile of the ROM on Github. I'm not sure if this is helpful, but I found the function that is used to draw glyphs.

https://github.com/FireEmblemUniverse/fireemblem8u/blob/master/src/fontgrp.c#L1015

**Comment by RobertPeip** [comment]
That's a very good find, but i doubt it will change anything. I debugged into that a while ago on my emulator, which doesn't have this bug, and made changes to see what could be the reason.

I found out, that not writing the last byte(bit 31..24) of the dword access to VRAM leads to equal effect, but for ALL glyphs, not just the buggy ones. So there must be some kind of wrong memory access or cpu miscalculation earlier leading to this effect.

As there are still some CPU bugs pending(see the list of games not running), i guess this will be gone sometime when another CPU bug is fixed.
